Progress scaffold & implementation (#24023)
microsoft/fluentui: packages/react-components/react-progress/src/stories/Progress/ProgressDescription.md

> A progress alerts a user that content is being loaded or processed and they should wait for the activity to complete.

Reviewer: ```suggestion
A Progress provides a visual representation of content being loaded or processed.
```

This could be the description I was talking about in the Progress.tsx comment.
